jamaljsr/polar
# fix(network): display error details when a node fails to start

Closes #221 

### Description

Prior to this update, nodes that failed to start would show an error status but did not display an error message. Now the message is displayed in the sidebar. 

### Steps to Test

1. create a new network
2. start it one time successfully
3. click on one of your LND nodes and copy the path to the TLS cert file
4. stop the network and shutdown the app
5. add some garbage text to the TLS cert file
6. open the app and start the network
7. confirm that an error message is displayed

### Screenshots

<img width="1351" alt="Screen Shot 2019-10-26 at 11 04 14 PM" src="https://user-images.githubusercontent.com/1356600/67629712-71184980-f850-11e9-98e0-e6a57c97f89d.png">
